features/analytics.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Analytics
```

Comment:
I'm really not sure about this...
1. Not sure if these tests should be standalone... or if they should be integrated into other features? Analytics has a different user need... but is a separate feature or a bolt on?

Reply:
Think these are fine for now - We've used this pattern in other apps